help/status-and-availability.md

```diff
@@ -32,6 +32,16 @@ You can set a status emoji, status message, or both.
 
 1. Click **Save**.
 
+{tab|via-personal-settings}
+
+1. Click your profile picture on the top right
+
+1. Click **Edit Status**
+
+1. Tap to select one of the most common statuses, *or* choose any emoji and/or write a short message.
```

Comment:
This should be the exact same instruction as in the block via-user-card section, right?